app/src/test/java/be/xplore/githubmetrics/app/IntegrationTest.java

```diff
@@ -29,15 +30,21 @@
         webEnvironment = RANDOM_PORT
 )
 @AutoConfigureObservability
-public class JobCountsIntegrationTest {
-
+class IntegrationTest {
```

Comment:
since there are two other comments might aswell change this to plural xD

Reply:
done